### ISSUE TITLE
Ability to change angle

### ISSUE THREAD

**Issue by visualcookie** [josephfusco/angled-edges#9]
Sometimes the designer changes the angle so it's not 10deg but 2 or 3deg.

**Comment by josephfusco** [comment]
Great suggestion. The ability to specify the angle rather than having the user calculate height + width would be a nice addition. Currently working on #8 to allow percentages, which this wouldn't really work with as the angle would change at different screen sizes, but maybe we can do something like

```scss
@include angled-edge('outside bottom', 'lower left', #00FF00, 2800, $angle:4deg);
```

where we specify both width and angle as an optional argument.

**Comment by visualcookie** [comment]
That would be amazing. Like that idea. :+1:

**Comment by josephfusco** [comment]
@visualcookie This feature is now available in v1.2-alpha on master! Do you mind testing it?

**Comment by josephfusco** [comment]
Thinking this over again, and while I would love to include an angle option as previously  discussed, it has proven to be inefficient due to the limitations of sass. Creating the functions needed to calculate this doubles the size of the partial just for one argument.

For the time being, the user can still achieve a specific angle they need by calculating it themselves. This will keep compile times down. I will consider this however in the future once when sass can handle more of these functions itself.

For reference, if you need a specific angle for the edge, the equation would be:
```
$height = tan($angle) * $width
```

I hope this helps.
